### DISCUSSION TITLE
Advanced Options not showing

### DISCUSSION THREAD

**Discussion by eggbean** [sumatrapdfreader/sumatrapdf#3749] [Q&A] [answered]
I am using v3.4.6 64-bit on Windows 11.

When I select Advanced Options from the menu or the Command Palette, nothing happens.

I would like to customise the keyboard shortcuts, but I cannot do this because of this issue.

**Comment by kjk** [comment, marked as answer]
Did you uninstall Window's built-in notepad? Do you have any text editor installed (Visual Studio Code, notepad++, textmate)?

We try to auto-detect a text editor and open settings file with it.

If that doesn't happen then either none of the editors we support is installed on your computer or there is some other reason we can't detect them. Or maybe editor installation is corrupted and cannot be launched.

Either way, you can edit settings file in any text editor. See https://www.sumatrapdfreader.org/docs/How-we-store-settings for where the file is.

You can also try upgrading to latest pre-release build: https://www.sumatrapdfreader.org/prerelease

**Comment by eggbean** [comment]
Oh, I see. It just opens that file.

I didn't uninstall Notepad, although I don't use it. IIRC, I did uninstall it previously a long time ago, but I had issues with other software that assumes that it is installed.

I use gVim and have all text filetypes associated with it. Thanks.
